test/functional/apps/dashboard_elements/controls/options_list.ts

```diff
@@ -28,13 +28,16 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     'header',
   ]);
 
+  const DASHBOARD_NAME = 'Test Options List Control';
+
   describe('Dashboard options list integration', () => {
     before(async () => {
       await common.navigateToApp('dashboard');
       await dashboard.gotoDashboardLandingPage();
       await dashboard.clickNewDashboard();
       await timePicker.setDefaultDataRange();
       await elasticChart.setNewChartUiDebugFlag();
+      await dashboard.saveDashboard(DASHBOARD_NAME, { exitFromEditMode: false });
```

Comment:
This bug was only present for existing dashboards, so need to save the dashboard before adding controls and testing to ensure that the `unsaved changes` badge can be cleared.